winrm.go

```diff
@@ -156,6 +171,7 @@ var defaultWinrmSettings winrmSettings = winrmSettings{
 	maxEnvelopeSize:  "153200",
 	locale:           "en-US",
 	operationTimeout: "PT60.000S",
+	isSecure:         true,
```

Comment:
comma is not required as isSecure is last attribute in this json.

Reply:
> comma is not required as isSecure is last attribute in this json.

it will give error.